commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -409,6 +409,7 @@ def run(args):
         print(terminate_instances_result)
         print(instance_ids)
 
+
 class Stop(object):
     command = 'stop'
     help = """Stop instances for a given env"""
```

Comment:
Does "instance" mean server?

Reply:
Yeah, it means specifically "EC2 instance", since this is all AWS-specific